### ISSUE TITLE
uncaught error with invalid model

### ISSUE THREAD

**Issue by boesi** [ember-bootstrap/ember-bootstrap-changeset-validations#58]
Hi

If I put an invalid value in my form, I get a nice error message in my template and the submit function isn't called. But I also get an error message in the developer console:
```
bs-form.js:28 Uncaught (in promise) Error
    at BsFormWithChangesetValidationsSupport.validate (bs-form.js:28:1)
validate                            @ bs-form.js:28
Promise.then (async)
iterations                          @ backburner.js:18
[...stack trace...]
```
How can I catch this error? And why do I even have to catch this error? Or do I miss something else?

Thanks for your help and all your work

**Comment by boesi** [comment]
I've found the cause: I want a form, that is automatically submitted, when the input field loses focus. 

For this I've build the form this way:
```
<BsForm @model={{changeset this.item this.validations}} @onSubmit={{this.saveItem}} as |form|>
    <form.element @property=... as |el|>
        <el.control {{on "focusout" form.submit}}/>
    </form.element>
</BsForm>
```
If I use a regular submit button, I don't get the error message in the console. 

I could call another action in the focusout event handler, and check myself if the model is valid:
```
        <el.control {{on "focusout" (fn this.otherAction ...)}}/>
```
... but how do I get the changeset?

If I'm interpreting the [documentation for the submit action](https://www.ember-bootstrap.com/api/classes/Components.Form.html#method_submit) correctly, my approach should (or could?) work. Ember-Changeset would be pretty ideal for this "no-save-button-style".

**Comment by jelhan** [comment]
> If I'm interpreting the [documentation for the submit action](https://www.ember-bootstrap.com/api/classes/Components.Form.html#method_submit) correctly, my approach should (or could?) work.

Yes. It should work. Do you have time to investigate why it is not working? A reproduction would be very helpful to understand what is happening. Also would be interesting to know if it is specific to Ember Bootstrap Changeset Validations. Or if it's a general issue with Ember Bootstrap's validation API. And if it's happening always if using `form.submit` or specific to timing of modifiers / runloop. To be honest I have no clue yet, what might happen.

**Comment by boesi** [comment]
I have created a simple test repository: [Auto Submit](https://github.com/boesi/auto-submit)

In my actual project I'am still using Bootstrap 4, but as you can see Bootstrap 5 shows the same error.

**Comment by jelhan** [comment]
This might be related to #61
